api/graphql/reservations/reservation_serializers.py

```diff
@@ -215,6 +215,7 @@ def validate(self, data):
             self.check_max_reservations_per_user(
                 self.context.get("request").user, reservation_unit
             )
+            data["sku"] = reservation_unit.sku
```

Comment:
Perhaps we should add a check that there aren't more `sku`s than one in the reservation's reservation units

Reply:
Good idea. I was trying to think what would happen if there is more than one reservation unit, but couldn't decide on the right approach. Maybe raising a validation error is the way to go here.

Reply:
I added validation for this, and also a test case that makes sure it works. This required some changes to the mock opening hours to allow for multiple reservation units.